data_structures/meta_data.py

```diff
@@ -114,3 +114,21 @@ def WriteMetaFile(self):
                 # need to call the function here so that we get the grid size if there was non in the original meta file
                 (crop_zmin, crop_zmax), (crop_ymin, crop_ymax), (crop_xmin, crop_xmax) = self.CroppingBox()
                 fd.write('{}:{}x{}:{}x{}:{}\n'.format(crop_xmin, crop_xmax, crop_ymin, crop_ymax, crop_zmin, crop_zmax))
+
+def WriteBossMetaFile(coord_frame, cutout, prefix):
+    meta_path = os.path.dirname(os.getcwd()) + '/neuronseg/meta'
```

Comment:
Would these relative paths stay consistent in a docker image? I'm imagining yes but want to make sure.